### PR TITLE
add openkimmodels

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jan-janssen** [comment]
Inspired by https://github.com/Homebrew/homebrew-core/blob/master/Formula/openkim-models.rb

**Comment by jan-janssen** [comment]
I currently struggle to configure the compilers for cMake: 

```
CMake Error at CMakeLists.txt:37 (project):
  The CMAKE_Fortran_COMPILER:

    /home/conda/feedstock_root/build_artifacts/kim-api_1571991449443/_build_env/bin/x86_64-conda_cos6-linux-gnu-gfortran

  is not a full path to an existing compiler tool.
```

**Comment by jan-janssen** [comment]
> I currently struggle to configure the compilers for cMake:
> 
> ```
> CMake Error at CMakeLists.txt:37 (project):
>   The CMAKE_Fortran_COMPILER:
> 
>     /home/conda/feedstock_root/build_artifacts/kim-api_1571991449443/_build_env/bin/x86_64-conda_cos6-linux-gnu-gfortran
> 
>   is not a full path to an existing compiler tool.
> ```

Fixed with https://github.com/conda-forge/staged-recipes/pull/10039/commits/75f42cf2dfd3433d6c595df70e8b5a672158775f

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/openkimmodels/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "openkimmodels" %}
+{% set version = "2019.07.25" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://s3.openkim.org/archives/collection/openkim-models-2019-07-25.txz
```
Comment (chrisburr) [inline recipes/openkimmodels/meta.yaml]:
```suggestion
  url: https://s3.openkim.org/archives/collection/openkim-models-{{ version|replace('.', '-') }}.txz
```

**Comment by jan-janssen** [comment]
> In case you didn't figure out why, `$CXX`/`$CC`/... are needed as the compiler binaries are prefixed with the platform to prevent the wrong ones from being used.
> 
> Have you tried building for macOS? It normally works out of the box using the Linux and is nice to have even if it isn't a priority. If nothing else it occasionally finds bugs that affect Linux but don't crash.

I tried Mac OS in https://github.com/conda-forge/staged-recipes/pull/10039/commits/69945f756d586d5273193364a33eb2264a92c465 but it does not work with clang, seems like gcc is required.

**Pull request merged**